src/index.ts

```diff
@@ -57,3 +60,29 @@ ${exports
     }
   };
 }
+
+async function getImportUrl(id: string, from: string, moduleIds: string[]) {
+  const importerPath = id.slice(0, id.lastIndexOf("/")) + from.slice(from.lastIndexOf("/"));
```

Comment:
Consider replace it with:

```suggestion
  const importerPath = path.resolve(id, "../" + from);
```